Microsoft.Toolkit.Uwp.UI.Controls/FlexPanel/FlexPanel.cs

```diff
@@ -0,0 +1,1613 @@
+
+            /// <summary>Gets or sets  the left edge absolute position of the item.It also defines the item's width if <see cref="FlexItem.Right" /> is also set and if <see cref="FlexItem.Width" /> isn't set.It is ignored if <see cref = "FlexItem.Position" /> isn't set to Absolute.</summary>
+            /// <value>The value for the property.</value>
+            /// <remarks>The default value for this property is NaN.</remarks>
+            public double Left { get; set; } = double.NaN;
+
+            /// <summary>Gets or sets  the margin space required on the bottom edge of the item.</summary>
+            /// <value>The top edge margin space (negative values are allowed).</value>
+            /// <remarks>The default value for this property is 0.</remarks>
+            public double MarginBottom { get; set; } = 0f;
+
+            /// <summary>Gets or sets  the margin space required on the left edge of the item.</summary>
+            /// <value>The top edge margin space (negative values are allowed).</value>
+            /// <remarks>The default value for this property is 0.</remarks>
+            public double MarginLeft { get; set; } = 0f;
+
+            /// <summary>Gets or sets  the margin space required on the right edge of the item.</summary>
+            /// <value>The top edge margin space (negative values are allowed).</value>
+            /// <remarks>The default value for this property is 0.</remarks>
+            public double MarginRight { get; set; } = 0f;
+
+            /// <summary>Gets or sets  the margin space required on the top edge of the item.</summary>
+            /// <value>The top edge margin space (negative values are allowed).</value>
+            /// <remarks>The default value for this property is 0.</remarks>
+            public double MarginTop { get; set; } = 0f;
+
+            private int order = OrderDefault;
+
+            /// <summary>Gets or sets whether this item should be laid out before or after other items in the container.Items are laid out based on the ascending value of this property.Items that have the same value for this property will be laid out in the order they were inserted.</summary>
+            /// <value>The item order (can be a negative, 0, or positive value).</value>
+            /// <remarks>The default value for this property is 0.</remarks>
+            public int Order
+            {
+                get => order;
+                set
+                {
+                    if ((order = value) != 0 && Parent != null)
+                    {
+                        Parent.ShouldOrderChildren = true;
+                    }
+                }
+            }
+
+            /// <summary>Gets or sets the height of the item's bottom edge padding space that should be used when laying out child items.</summary>
+            /// <value>The bottom edge padding space.Negative values are not allowed.</value>
+            public double PaddingBottom { get; set; } = 0f;
```

Comment:
You can use the Thickness object for all four of these properties and label it Padding
`public Thickness Padding { get; set; }`